src/integration/billing/BillingIntegration.ts

```diff
@@ -177,27 +179,50 @@ export default abstract class BillingIntegration {
       inError: 0
     };
     await this.checkConnection();
-    const openedInvoices = await BillingStorage.getInvoicesToPay(this.tenantID);
-    // Let's now try to pay opened invoices
-    for (const openInvoice of openedInvoices.result) {
+
+    let invoices: DataResult<BillingInvoice>;
+    if (this.settings.billing?.periodicBillingAllowed) {
+      // Fetch DRAFT and OPEN invoices
+      invoices = await BillingStorage.getInvoicesToProcess(this.tenantID);
+    } else {
+      // Fetch OPEN invoices only
+      invoices = await BillingStorage.getInvoicesToPay(this.tenantID);
+    }
+    // Let's now finalize all invoices and attempt to get it paid
+    for (const invoice of invoices.result) {
       try {
-        await this.chargeInvoice(openInvoice);
+        // Make sure to avoid trying to charge it again too soon
+        if (moment(invoice.createdOn).isSame(moment(), 'day')) {
+          actionsDone.inSuccess++;
```

Comment:
You count not synced right now as success? Why not count is them with a specific counter like postPostponed or skipped.